src/core/tests/type_prop/search_sorted.cpp

```diff
@@ -93,6 +97,19 @@ TEST(type_prop, search_sorted_shape_infer_both_dynamic_9) {
     PerformShapeTest({-1, -1}, PartialShape::dynamic(), {-1, -1});
 }
 
+TEST(type_prop, search_sorted_shape_symbols) {
+    PartialShape sorted_shape{1, 3, 7, 100};
+    PartialShape values_shape{-1, -1, -1, 10};
+    auto sorted_symbols = set_shape_symbols(sorted_shape);
+    auto values_symbols = set_shape_symbols(values_shape);
+    auto sorted = make_shared<op::v0::Parameter>(element::i32, sorted_shape);
+    auto values = make_shared<op::v0::Parameter>(element::i32, values_shape);
+    auto search_sorted_op = make_shared<op::v15::SearchSorted>(sorted, values);
+    EXPECT_EQ(search_sorted_op->get_element_type(), element::i64);
+    EXPECT_THAT(get_shape_symbols(search_sorted_op->get_output_partial_shape(0)),
```

Comment:
In specification (#26887) output description:
 ` with the same shape as second input tensor`, 
 the symbols of 2nd input should  have priority over symbols from 1st input.

Reply:
Fixed.